### ISSUE TITLE
feature request: draw triangle (group 5)

### ISSUE THREAD

**Issue by zbyti** [paulscottrobson/neo6502-firmware#605]
a filled triangle as an option would be nice ;)

**Comment by paulscottrobson** [comment]
Good idea. On holiday next week so I'll look at this when I get back. 

Anyone who knows anything about triangle/polygon type algorithms and can recommend one ?

**Comment by zbyti** [comment]
https://cglearn.eu/pub/computer-graphics/task/bresenham-triangle-1

**Comment by paulscottrobson** [comment]
Not implemented for 1.0 , the primary reason is it will require some surgery to the BASIC drawing code which does not support more than 2 points.
